pyproject.toml

```diff
@@ -134,6 +136,10 @@ fixable = ["ALL"]
 exclude = [
     "cpp/scripts/gdb-pretty-printers.py",
 ]
+extend-unsafe-fixes = [
```

Comment:
IIUC from your [comment](https://github.com/rapidsai/cudf/issues/15835#issuecomment-3145124008) this for detection, we still need to manually set `strict=True`?

Reply:
Correct, the purpose of this setting is to allow ruff to at least error when strict is missing, while not autofixing with strict=False. The end result is requiring a manual fix.